security/s2a/crypter/record.go

```diff
@@ -0,0 +1,16 @@
+package crypter
+
+// S2AAeadCrypter is the interface for gRPC S2A record protocol.
+type S2AAeadCrypter interface {
+	// Encrypt encrypts the plaintext and computes the tag (if any) of dst
```

Comment:
Please remove `(if any)` and similarly in the comments of `Decrypt`. For our use cases, we will always have a tag.